src/styles/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  max-width: 1200px;
+  display: flex;
+  font-family: Roboto, system-ui;
```

Comment:
I would recommend moving the font-family to a more general selector, like body. This way, you ensure that the font-family is inherited by all elements unless otherwise specified.